README.md

```diff
@@ -249,13 +218,119 @@ Invoking the `eval` method of each dataset yields all metrics reported in the pa
+```
+Note that downloading the large amount of unlabeled data is optional; unlabeled data will only be downloaded if some `--unlabeled_split` is set. (It does not matter which `--unlabeled_split` is set; all unlabeled data will be downloaded together.)
+
+Alternatively, you can use the standalone `wilds/download_datasets.py` script to download the datasets, for example:
+
+```bash
+# downloads (labeled) data
+python wilds/download_datasets.py --root_dir data
+
+# downloads (unlabeled) data
+python wilds/download_datasets.py --root_dir data --unlabeled
+```
+
+This will download all datasets to the specified `data` folder. You can also use the `--datasets` argument to download particular datasets.
+
+These are the sizes of each of our datasets, as well as their approximate time taken to train and evaluate the default model for a single ERM run using a NVIDIA V100 GPU.
+
+| Dataset command | Modality | Download size (GB) | Size on disk (GB) | Train+eval time (Hours) |
+| --------------- | -------- | ------------------ | ----------------- | ----------------------- |
+| iwildcam        | Image    | 11                 | 25                | 7                       |
+| camelyon17      | Image    | 10                 | 15                | 2                       |
+| rxrx1           | Image    | 7                  | 7                 | 11                      |
+| ogb-molpcba     | Graph    | 0.04               | 2                 | 15                      |
+| globalwheat     | Image    | 10                 | 10                | 2                       |
+| civilcomments   | Text     | 0.1                | 0.3               | 4.5                     |
+| fmow            | Image    | 50                 | 55                | 6                       |
+| poverty         | Image    | 12                 | 14                | 5                       |
+| amazon          | Text     | 7                  | 7                 | 5                       |
+| py150           | Text     | 0.1                | 0.8               | 9.5                     |
+
+The following are the sizes of the unlabeled data bundles:
+
+| Dataset command | Modality | Download size (GB) | Size on disk (GB) |
+| --------------- | -------- | ------------------ | ----------------- |
+| iwildcam        | Image    | 41                 | 41                |
+| camelyon17      | Image    | 69.4               | 96                |
+| ogb-molpcba     | Graph    | 1.2                | 21                |
+| globalwheat     | Image    | 103                | 108               |
+| civilcomments   | Text     | 0.3                | 0.6               |
+| fmow\*          | Image    | 50                 | 55                |
+| poverty         | Image    | 172                | 184               |
+| amazon\*        | Text     | 7                  | 7                 |
+
+\* The unlabeled dataset is the same as the labeled dataset.
```

Comment:
Maybe we can elaborate on this a bit -- like we don't have to redownload the unlabeled dataset?